repos/system_upgrade/el7toel8/actors/grubdevname/actor.py

```diff
@@ -1,9 +1,14 @@
+import platform
+
 from leapp.actors import Actor
 from leapp.libraries.actor.grubdevname import get_grub_device
+from leapp.libraries.common.config import architecture
+from leapp.libraries.common.utils import skip_actor_execution_if
 from leapp.models import GrubDevice
 from leapp.tags import FactsPhaseTag, IPUWorkflowTag
 
 
+@skip_actor_execution_if(platform.machine() == architecture.ARCH_S390X)
```

Comment:
Read my commet about the decorator. Anyway, you should not use `platform.machine` anywhere. Use the configuration function: `architecture.matches_architecture(architecture.ARCH_S390X)`